.env

```diff
@@ -24,13 +24,13 @@ APP_SECRET=2a45d2e71fe1aa844fad8744ff9846a6
 # Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
 # For an SQLite database, use: "sqlite:///%kernel.project_dir%/var/data.db"
 # Configure your db driver and server_version in config/packages/doctrine.yaml
-DATABASE_URL=mysql://db_user:db_password@127.0.0.1:3306/db_name
+DATABASE_URL=mysql://root:root@127.0.0.1:3306/server1
 ###< doctrine/doctrine-bundle ###
 
 ###> oauth2/discord ###
 # See https://discordapp.com/developers/docs/topics/oauth2
-oauth_discord_id=
-oauth_discord_secret=
+oauth_discord_id=761278563722461234
```

Comment:
Hi be careful about committing your credentials. I suggest you to remove the file .env from this Pull Request.